### ISSUE TITLE
how to respond to mouse scroll?

### ISSUE THREAD

**Issue by jessitron** [jessitron/bird#1]
'cuz then we could zoom out instead of refreshing

@DamonOehlman

**Comment by DamonOehlman** [comment]
Yeah, the whole refreshing thing is kind of a drag... I'll do something in branch and send through a PR :)
